website/app/styles/doc-components/icons-list/index.scss

```diff
@@ -103,10 +109,36 @@
 }
 
 .doc-icons-list-grid-item__meta {
+  position: relative;
   width: 100%;
-  padding: 16px;
+  min-width: 0;
+  padding: 8px 16px;
 }
 
+.doc-icons-list-grid-item__permalink {
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 24px;
+  height: 24px;
+  color: var(--doc-color-gray-300);
+  visibility: hidden;
```

Comment:
This is likely to cause accessibility issues. (Tagging @MelSumner)

See: https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#accessibility_concerns

Reply:
Yeah, you're right, it was a sloppy implementation. I've pushed a change to fix it.